tests/deptest_routes.py

```diff
@@ -0,0 +1,265 @@
+import time
+
+from tests.helpers import count_fund_applications
+from tests.helpers import expected_data_within_get_response
+from tests.helpers import post_data
+
+
+# TODO: Not possible to test all applications currently as unpredictable UID in default data
```

Comment:
Have you created an extra story to address this if you're not going to address it in this PR?

Reply:
I'd prefer if you removed these and left them on a branch

Reply:
These are deprecated tests - removing